tower_modules/tower_license/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Set license file
+  tower_license:
+    tower_host: [TOWER HOST]
+    tower_username: [USERNAME]
+    tower_password: [PASSWORD]
+    validate_certs: False
+    data: "{{ lookup('template', '/Users/path_to_file/license.txt', convert_data=True) }}"
+    eula_accepted: True
```

Comment:
Oh I don't think we can accept this, because it's a new module. So let's delete it for now, pending some new structure for these tests.

Reply:
Hmm, i think we might actually be fine adding this, but we can always merge the other stuff first